src/Raven.Server/ServerWide/DebugInfoPackageUtils.cs

```diff
@@ -47,5 +49,26 @@ public static void WriteExceptionAsZipEntry(Exception e, ZipArchive archive, str
             }
         }
 
+        public static bool CanAccessRoute(RavenServer.AuthenticateConnection authenticateConnection, RouteInformation route, string db = null)
```

Comment:
This does not belong here. This is duplicate of already existing code, no?

Reply:
Removed and now using AuthenticateConnection.CanAccess() instead